tests/test_endpoints.py

```diff
@@ -0,0 +1,55 @@
+import requests
+import os
+from dotenv import load_dotenv
+from itertools import islice
+import pytest
+
+load_dotenv('./env')
+FLASK_PORT = int(os.getenv('FLASK_PORT'))
+
+
+def test_labels(file_path: str):
+    global FLASK_PORT
+
+    if not os.path.exists(file_path):
+        print(f"This file doesn't exist: {file_path}")
+        return dict()
+
+    url = f"http://localhost:{FLASK_PORT}/match"
+    response = requests.post(url, files={'file': open(file_path, 'rb')})
+
+    if response.status_code == 200:
+        outp_json = response.json()
+        print(f"Response json is: {outp_json}")
+    else:
+        outp_json = dict()
+        print(f"Request failed with status code: {response.status_code}; file path: {file_path}")
+
+    assert len(outp_json) > 0, "Empty json"
+    assert len(outp_json) == 29, "Wrong size json"
+
+    first_5_elements = dict(islice(outp_json.items(), 5))
+
+    expected_values = {
```

Comment:
What happen if zooma has new knowledge and there is another mapping in zooma output.

Reply:
possibly we can check json keys only, like:
'Age at present'
'Age at the agreement date',
etc. ...